test/config/config_test.go

```diff
@@ -0,0 +1,188 @@
+package config
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/regionless-storage-service/pkg/config"
+)
+
+func TestLocalStores(t *testing.T) {
```

Comment:
suggest one test case to test the function GetReplica...; also better not to use test data 127.x.x.x as non-local stores to avoid code reading confusions

Reply:
Ip address does not matter here since they are for testing only. I have updated them for less confusion.